README.md

```diff
@@ -94,7 +91,11 @@ Note that if you get errors, run the above commands with `--verbose` - this will
 
 ## Development
 
-This modules has been created for educational reasons, so any updates to the documentation are welcome.
+In order to aid in development a java based emulator is included within the `spec/fictures` folder, allowing you to more easily test and develop the code within this module. In order to start it up simply enter the following into the command line:
+```
+java -jar ./spec/fixtures/HueEmulator-v0.8.jar
```

Comment:
Please do not commit the JAR file into the repo, especially since there is no license on the original file. Instead link to the source repo.

Related: even we could directly license the file for redistribution, there still should be a link back to the source of the file.

Reply:
Have removed the .jar file and updated the README.md reference to include a link to the GitHub page.